### DISCUSSION TITLE
tooltip on button node is showed on window focus

### DISCUSSION THREAD

**Discussion by ijcnvv** [Akryum/v-tooltip#806] [Q&A]
lib version 2 and 3

1. options by default
2. click to some **button** node which has tooltip
3. click to another space to make brower tab inactive
4. click to browser tab to make it focus
5. tooltip is showed

https://codesandbox.io/s/falling-framework-cjccy?file=/src/main.js

**Comment by Akryum** [comment]
Its because of the focus trigger.
